app/controllers/compute_resources_controller.rb

```diff
@@ -36,6 +36,11 @@ def create
       @compute_resource.valid?
       process_error
     end
+  rescue Fog::Ovirt::Errors::OvirtEngineError => e
+    logger.error "Error while creating ovirt resource: #{e.message}\n#{e.backtrace.join("\n")}"
```

Comment:
There is a logger helper that can/should be used, see for example https://github.com/theforeman/foreman/blob/4a96211d33c261655c887bbb6c787e38d883200a/app/validators/ssh_key_validator.rb#L13.

Reply:
Also, there is ovirt mentioned here as well, while it should not (as well as in other places of this PR)

Reply:
I'm missing here something in my understanding.
The original issue was, that when exception was raised, the actual 500 page would have shown rather then an error message the "foreman way".

My original attempt was to make it display the "foreman way", so if I create a validator here, how would it solve the problem of having the 500 error page?

Having said that, I do understand that the controller is agnostic to the worker that doing the actual task, and will solve this.

Reply:
No, I haven't meant validator: it was just an example of logging the exception with the `Logging.exception` API

Reply:
Btw. since we rescue `StandardError` at the other place, I wonder, if we shouldn't use this here as well compared to `Fog::Ovirt::Errors::OvirtEngineError`

Reply:
The original code used `StandardError`, and moved it to use `Fog::Ovirt::Errors::OvirtEngineError` following the above discussion,

Reply:
I'm ok with that then. Still asking for using `Foreman::Logging.exception` use here